src/lib/Scenes/City/City.tsx

```diff
@@ -179,7 +179,6 @@ export class CityView extends Component<Props, State> {
             <ScrollableTabView
               initialPage={this.props.initialTab || AllCityMetaTab}
               onChangeTab={this.setSelectedTab}
-              prerenderingSiblingsNumber={2}
```

Comment:
Won't this remove the optimization of loading the next tabs for the user?

Reply:
Yes it will but now that we drastically reduced rendering tab of list view (capped to 30 items), it's not needed anymore as they load fast enough for the user not to notice

Reply:
Pretty good reason to remove it!